test/e2e/downwardapi_volume.go

```diff
@@ -32,18 +30,6 @@ import (
 // How long to wait for a log pod to be displayed
 const podLogTimeout = 45 * time.Second
 
-// utility function for gomega Eventually
-func getPodLogs(c *client.Client, namespace, podName, containerName string) (string, error) {
```

Comment:
Ignore my above comment. I thought it is test/e2e/util.go. I saw you moved it to the right file below.